### ISSUE TITLE
Possible issue that is creating a fatal exception

### ISSUE THREAD

**Issue by Will-Nichols** [ChrisAdderley/StationPartsExpansionRedux#227]
I am getting this error now and am not sure if it is from Station Parts Expansion Redux or B9 Parts Switcher. After getting these initial screenshots I loaded without Station Parts Expansion Redux and didn't get a fatal exception error.
https://gyazo.com/a0e261835483a9248978f06f4644e1c9

https://gyazo.com/e2710009f747c0eb0feaf4f904355499

Reloaded the mod after checking it and generated a visible player log file. Here is the link to the [log file](https://www.dropbox.com/s/j9p4uzcj66w04s2/Player.log?dl=0).

Also here is a screenshot of the modlist I am loading. 
https://gyazo.com/725a212d99a6191cda69720670b1a8f5

**Comment by Will-Nichols** [comment]
Wanted to link this to here just in case these are somehow connected from the [B9 Parts Switcher mod](https://github.com/blowfishpro/B9PartSwitch/issues/191#event-3217130175).

**Comment by ChrisAdderley** [comment]
Ok, there are a shocking amount of exceptions in that log. 

However, filtering through, that log is showing that the `Silicates` resource does not exist - the resource definition is failing to load. 
Now, Silicates is a CRP resource that exists here:
https://github.com/BobPalmer/CommunityResourcePack/blob/master/FOR_RELEASE/GameData/CommunityResourcePack/CommonResources.cfg#L1424
I'm not sure what would cause it not to load. Either
1. That definition is missing from the file (recommend you check it in the CRP/CommonResources config file)
2. Some other mod has removed the definition from the game

Besides that, MKS is generating tons of other exceptions and overall, weird stuff is going on.

**Comment by Will-Nichols** [comment]
OK I think I will pull MKS from the install and see what is happening. I just posted on the KSP forums about a [missing part](https://forum.kerbalspaceprogram.com/index.php?/topic/193007-has-anyone-else-had-missing-stock-parts-after-reinstalling-fresh/) as well. I reinstalled KSP yesterday and its apparently trying to revert to pre-1.9.x from an old savegame. I have no idea how or why but things like Mechjeb and other flight assist mods are all kinds of jacked up.
